docs/source/docs/installation.blade.md

```diff
@@ -157,12 +157,12 @@ If you're using a preprocessor, use the `options` method to add `tailwindcss` as
 ```js
 var tailwindcss = require('tailwindcss');
 
-mix.less('source/_assets/less/main.less', 'source/css')
+mix.less('resources/assets/less/app.less', 'public/css')
```

Comment:
I would go further and even change app.less to app.css, as you can use plain css with Tailwind.

Reply:
While it's true that you might not need a preprocessor with the way that Tailwind works, this example is specific to using Less. And there is already a plain css example above this one.

Reply:
Ach, gotcha!